.storybook/components/DataTable/DataTableRenderers/EditableTextRenderer.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Input from '@airbnb/lunar/src/components/Input';
+import Text from '@airbnb/lunar/src/components/Text';
+
+const onClick = e => e.stopPropagation();
+
+class EditableTextRenderer extends React.Component {
+  state = {
+    value: this.props.value,
+  };
+
+  onEdit = (row, key) => (newVal, event) => {
+    const { onEdit } = this.props;
+    onEdit(row, key, newVal, event);
+
+    this.setState({
+      value: newVal,
+    });
+  };
+
+  render() {
+    const { editMode, row, key } = this.props;
+    const { value } = this.state;
+
+    return editMode ? (
+      <Input
+        label=""
```

Comment:
how about a label like "Edit row"? Maybe with the row number? having a label is important for accessibility, even if it is visually hidden